agasc/agasc.py

```diff
@@ -157,6 +186,7 @@ def get_agasc_cone(ra, dec, radius=1.5, date=None, agasc_file=None,
     :param date: Date for proper motion (default=Now)
     :param agasc_file: Mini-agasc HDF5 file sorted by Dec (optional)
     :param pm_filter: Use PM-corrected positions in filtering
+    :param fix_color1: set COLOR1=COLOR2 * 0.85 for stars with V-I color
```

Comment:
Are all of our COLOR2 values under 2.0 or some such?  I thought there were some limits on the range for which the 0.85 conversion was good, but may not apply to our set.

Reply:
I don't know the range of COLOR2.  The TYCHO2 readme doesn't indicate any range of applicability, though I didn't read the paper they reference for details.  In any case it doesn't change anything for downstream code or analysis.

Bt_Mag
The Tycho-2 BT magnitude, blank when no magnitude is available. Either BT (bt_mag) or VT (vt_mag) is always given. Approximate Johnson photometric magnitudes may be obtained as follows:

      V   = VT - 0.090*(BT-VT)
      B-V = 0.850*(BT-VT)
  
Consult Sect 1.3 of Vol 1 of "The Hipparcos and Tycho Catalogues", ESA SP-1200, 1997, for more details.

Reply:
Google had given me the Tycho Introduction and Guide to the Data Pg 57

https://www.cosmos.esa.int/documents/532822/552851/vol1_all.pdf/99adf6e3-6893-4824-8fc2-8d3c9cbba2b5

![image](https://user-images.githubusercontent.com/791508/49080257-0e1dcd00-f211-11e8-8a27-846271c1d8e3.png)

Reply:
Not that it matters much for our uses.